src/tool_help.c

```diff
@@ -192,6 +192,8 @@ static const struct helptxt helptext[] = {
    "Allow insecure server connections when using SSL"},
   {"    --interface <name>",
    "Use network INTERFACE (or address)"},
+  {"    --localaddr <IP-ADDR>",
+   "Specify local IP-Address to use"},
```

Comment:
There needs to be a `docs/cmdline-opts/localaddr.d` file too that documents this option.